docs/about.rst

```diff
@@ -4,25 +4,26 @@ About Weblate
 Project goals
 -------------
 
-Web based translation tool with tight git integration supporting a wide range
+Web based translation tool with tight Git integration supporting a wide range
 of file formats, making it easy for translators to contribute.
 
 The translations should be kept within the same repository as the source code and
-the translation process should closely follow development.
+the translation process should follow development closely.
 
-There is no plan regarding heavy conflict resolution as these should be primarily
-handled by git.
+There is no plan regarding heavy conflict resolution, as these should be
+handled primarily by Git.
 
 Project name
 ------------
 
-The project is named as mixture of the words web and translate.
```

Comment:
On [Wikipedia](https://en.wikipedia.org/wiki/Weblate) this was changed to "portmanteau", though it's probably something most people won't understand...